### PR TITLE
Allow operations on felixconfigurations in tigera clusterroles in ord…

### PR DESCRIPTION
**Pull request by rene-dekker**
Allow operations on felixconfigurations in tigera clusterroles in order to support UI features.

### CODE REVIEW

File: pkg/render/apiserver.go
Comment (caseydavenport) [inline pkg/render/apiserver.go:1772]: This does more than just read - is the comment correct or is the code correct?

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Review by caseydavenport — APPROVED** [review]

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
